nautobot/core/templates/utilities/object_bulk_create.html

```diff
@@ -0,0 +1,147 @@
+{% extends 'base.html' %}
```

Comment:
How do these new generic templates differ from the corresponding existing generic templates? If they're identical, can we get away with symlinks instead of duplicating the files entirely in git?